autogluon/training/docker/0.2/py3/cu102/Dockerfile.gpu

```diff
@@ -41,6 +41,16 @@ RUN apt-get update \
  && rm -rf /var/lib/apt/lists/* \
  && rm -rf /tmp/*
 
+# install OpenSSL
```

Comment:
Is there a reason we can't install openSSL from apt-get?

Reply:
ditto